lib/reporters/html/partials/aggregations.hbs

```diff
@@ -0,0 +1,56 @@
+{{#if folder}}
```

Comment:
While I know that it's generally a good practice to separate templates out into partials, it's unnecessary in this case, because it is being used only in one place. 

@kunagpal Unless there's any other reason you've made this change, I think we should keep the template to a single handlebars file.